pontis-ui/src/components/Landing/Testimonial.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+const Testimonial = () => (
+  <div className="relative w-full max-w-3xl lg:flex lg:items-center">
+    <svg
```

Comment:
Move this into it's own file? Or have inline svg? (These are the quotation marks behind the quote)